CONTRIBUTING.md

```diff
@@ -1,15 +1,9 @@
 Before contributing you should sign [Canonical's contributor agreement](http://www.ubuntu.com/legal/contributors), it’s the easiest way for you to give us permission to use your contributions.
 
-## Pull Request management
+## Pull Requests and tests
 
-Before merging any pull request to snappy's code base we need to verify that the code functionality and quality is not degraded by that addition. In order to do that there's a set of checks that we run for each PR, some of them using external services (TravisCI and Coveralls) and others using our own CI infrastructure (integration tests and autopkgtests). The checks based on external services are run for all the pull request. We are using the [GitHub Pull Request Builder Plugin](https://github.com/jenkinsci/ghprb-plugin/blob/master/README.md) for easing the management of PRs in relation with our internal infrastructure.
+We need to verify that the code functionality and quality is not degraded by additions before merging any changes to snapd's codebase. For each PR we run checks in three different groups: static, unit and spread.
 
-Depending on the affiliation of the GitHub user submitting a PR, the following actions may happen after receiving it:
+Static test use several code analysis tools present in the GoLang ecosystem (go vet, go lint and go fmt) to make sure that the code always aligns with the standards. They also check the markdown format of documentation files. All the existing unit tests are also executed, and the coverage info is reported to coveralls. Regarding [spread](https://github.com/snapcore/spread), we use it to verify the integrity of the product exercising it as a whole, both from an end user standpoint (eg. all kind of interactions with the snap tool from the command line) and from a more systemic approach (testing upgrades, for instance).
```

Comment:
can we wrap these at ~70, please? makes reading the raw text a lot easier, and doesn't affect the rendered output